internal/mode/static/state/dataplane/configuration.go

```diff
@@ -472,11 +472,14 @@ func buildUpstreams(
 	ctx context.Context,
 	listeners []*graph.Listener,
 	resolver resolver.ServiceResolver,
+	baseHTTPConfig BaseHTTPConfig,
```

Comment:
since buildUpstreams doesn't need the whole BaseHTTPConfig, could we only pass `IPFamilyType` ?